evaluations/src/lib.rs

```diff
@@ -203,12 +341,11 @@ pub async fn run_evaluation(
                     evaluation_name: &evaluation_name,
                     function_config: &function_config,
                     input: &input,
-                    inference_cache: args.inference_cache,
+                    inference_cache,
                 })
-                .await.map_err(|e| anyhow!("Error inferring for datapoint {datapoint_id}: {e}"))?,
```

Comment:
why are we no longer doing this? it is helpful to have the information for debugging